scopesim/optics/image_plane_utils.py

```diff
@@ -858,3 +933,20 @@ def split_header(hdr, chunk_size, wcs_suffix=""):
             hdr_list.append(hdr_sky)
 
     return hdr_list
+
+
+def _fix_360(arr):
+    """Fix the "full circle overflow" that occurs with deg."""
+    arr = np.asarray(arr)
+    arr[arr > 270] -= 360
+    arr[arr <= -90] += 360
+    return arr
```

Comment:
Maybe `_fix_360()` should have a while loop in it or something for values of e.g. 1000 and -1000. But we can do that when we need it.

Reply:
I already have a note somewhere saying this function should be looked at again 😅 
But the cases where this is needed are so far all "overflow" over zero, i.e. you get something like 359.995 instead of -0.005. And those are all caught by this. More interesting is what can happen if we're in arcsec instead of deg...